srcpkgs/circleci-cli/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'circleci-cli'
+pkgname=circleci-cli
+version=0.1.9578
+revision=1
+build_style=go
+go_import_path=github.com/CircleCI-Public/${pkgname}
+go_ldflags="-s -w -X github.com/CircleCI-Public/${pkgname}/version.Version=${version} -X github.com/CircleCI-Public/${pkgname}/version.Commit=cf6a918 -X github.com/CircleCI-Public/${pkgname}/version.packageManager=xbps"
+hostmakedepends="packr"
+short_desc="Use CircleCI from the command line"
+maintainer="Gabriel Sanches <gabriel@gsr.dev>"
+license="MIT"
+homepage="https://circleci-public.github.io/circleci-cli/"
+distfiles="https://github.com/CircleCI-Public/${pkgname}/archive/v${version}.tar.gz"
+checksum=d5dac07d27d97ee41dced89cf2b36dfeea17906343b6a6a21c1d8c4ca34a78a0
+
+pre_build() {
+	packr
+}
+
+post_build() {
+	packr clean
+}
+
+pre_install() {
+	mv ${GOPATH}/bin/circleci{-cli,}
```

Comment:
Why change the command name? Will that not cause confusion?

Reply:
CircleCI itself builds and distributes the CLI as `circleci` by passing the `-o circleci` flag to `go build`. Since I'm using the Go build style to run `do_build`, it won't let me pick the binary name and will simply follow the package name. Then I just rename it to follow the official distribution from CircleCI.

Reply:
I see. Makes sense to use the name that upstream uses.

Reply:
Looks like this is causing issues because the ARM bin dirs are different from the x86 ones. You should instead switch to doing the mv in post_install: `mv ${DESTDIR}/usr/bin/circleci{-cli,}`